### ISSUE TITLE
Version 1.0.0

### ISSUE THREAD

**Issue by demands** [goodeggs/resource-schema#15]
Seems like we're using this module pretty heavily in production already, should we do a 1.0.0 release?

**Comment by dannynelson** [comment]
@demands Yes! We definitely should. Next feature we add, we can bump it to 1.0.0.
